packages/react-aria-components/docs/Form.mdx

```diff
@@ -162,61 +162,25 @@ See the [Forms](forms.html) guide to learn more about form validation in React A
 
 ### Validation behavior
 
-By default, native HTML form validation is used to display errors and block form submission.
+By default, native HTML form validation is used to display errors and block form submission. To instead use ARIA attributes for form validation, set the `validationBehavior` prop to "aria". This will not block form submission, and will display validation errors to the user in realtime as the value is edited.
+
+The `validationBehavior` can be set at the form level to apply to all fields, or at the field level to override the form's behavior for a specific field.
 
 ```tsx example
-<Form>
-  <TextField name="email" type="email" isRequired>
-    <Label>Email</Label>
+<Form validationBehavior="aria">
+  <TextField
+    name="username"
+    defaultValue="admin"
+    isRequired
+    validate={value => value === 'admin' ? 'Nice try.' : null}>
+    <Label>Username</Label>
     <Input />
     <FieldError />
   </TextField>
-  <div style={{display: 'flex', gap: 8}}>
-    <Button type="submit">Submit</Button>
-    <Button type="reset">Reset</Button>
-  </div>
+  <Button type="submit">Submit</Button>
 </Form>
 ```
 
-To instead use ARIA attributes for form validation, set the validationBehavior prop to "aria". This will not block form submission, and will ensure validation errors are displayed to the user in realtime as the value is edited.
-
-This can be set at the form level to apply to all fields, or at the field level to override the form's behavior for a specific field.
-
-```tsx example
-const EMAIL_REGEX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
-
-function Example() {
-  let [value, setValue] = React.useState('');
-  return (
-    <Form validationBehavior="aria" onSubmit={(e) => {e.preventDefault()}}>
-      <TextField
-        name="email"
-        type="email"
-        value={value}
-        onChange={setValue}
-        isRequired
-        isInvalid={value.length > 0 && !EMAIL_REGEX.test(value)}>
-        <Label>Email</Label>
-        <Input />
-        <FieldError />
-      </TextField>
-      <div style={{display: 'flex', gap: 8}}>
-        <Button type="submit">Submit</Button>
-        <Button onPress={() => setValue('')}>Reset</Button>
-      </div>
-    </Form>
-  );
-}
-```
-
-The `validationBehavior` for a Form can be accessed from within any component inside a given From by using the `FormContext`.
-
-```tsx
-import {FormContext} from 'react-aria-components';
-
-let {validationBehavior} = useContext(FormContext);
-```
-
```

Comment:
Removed default example since we already have one above showing native validation. Adjusted the aria validation example to show a simpler custom validation example. Not sure we should recommend people validate emails via regex. Moved the context example to advanced customization below.